imod/prepare/wells.py

```diff
@@ -25,21 +25,42 @@ def vectorized_overlap(bounds_a, bounds_b):
     )
 
 
+def compute_point_filter_overlap(bounds_wells, bounds_layers):
+    """
+    Special case for filters with zero filter length, these are set to layer
+    thickness.
+    """
+    zero_filter_length = bounds_wells[:, 1] == bounds_wells[:, 0]
+    in_layer = (bounds_layers[:, 1] >= bounds_wells[:, 1]) & (
```

Comment:
Could you unwrap the bounds_layer and bounds_wells arrays? Im guessing one of them corresponds to the tops and the other to the bottoms

Reply:
Thanks! This makes it way easier for me to follow the logic